src/main/java/th/ac/kmitl/science/comsci/example/models/Seller.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package seller;
+
+/**
+ *
+ * @author Mirana
+ */
+public class Seller {
+
+    /**
+     * @param args the command line arguments
+     */
+    
+    private String sellerID;
```

Comment:
in Java variables naming convention, we preferred Lower Camel Case for variables name like:
```java
sellerID -> sellerId
sellerGlobalID -> sellerGlobalId
```